ethpm_types/sourcemap.py

```diff
@@ -0,0 +1,162 @@
+from typing import Dict, Iterator, List, Optional, Union
+
```

Comment:
I had to move this class so it could be imported from multiple places without conflict